certbot/display/ops.py

```diff
@@ -212,11 +212,13 @@ def success_installation(domains):
 
     """
     z_util(interfaces.IDisplay).notification(
-        "Congratulations! You have successfully enabled {0}{1}{1}"
+        "Congratulations! You have successfully {3}ed a certificate for{1}"
```

Comment:
You also should probably remove the newline here after "certificate for". The notification method being called here will handle creating newlines when necessary.